### DISCUSSION TITLE
reference settings.json

### DISCUSSION THREAD

**Discussion by e8vww** [transmission/transmission#6374] [Q&A]
I am updating from Transmission 3 to 4. It looks like there are some settings.json options that have been deprecated. There are probably some new ones too that I just missed.

Is there a reference file somewhere that I can compare with to make sure I'm following current best practices?

**Comment by LucaZeta** [comment]
I guess it's possible you'll find [such page ](https://github.com/transmission/transmission/blob/main/docs/Editing-Configuration-Files.md)interesting.
